
float32 N_dot
float32 E_dot
float32 d_dot


float32 u_dot
float32 v_dot
float32 w_dot



float32 p_dot
float32 q_dot
float32 r_dot


